.github/workflows/development-extension.yml

```diff
@@ -13,6 +13,10 @@ env:
   SEGMENT_WRITE_KEY: ${{ secrets.SEGMENT_WRITE_KEY_STAGING }}
   TRANSAK_API_KEY: ${{ secrets.TRANSAK_API_KEY }}
   BESTINSLOT_API_KEY: ${{ secrets.BESTINSLOT_API_KEY }}
+  BITFLOW_API_HOST: ${{ secrets.BITFLOW_API_HOST }}
```

Comment:
I wonder if I needed to add the `" "` around the value in GH secrets? I didn't so that might be why not working?

Reply:
Maybe I'll change and rerun the tests to see if they pass.

Reply:
That isn't right but I wonder if the urls are running into this? https://www.ssw.com.au/rules/handle-special-characters-on-github/

Do we need to keep the urls private, or can those just be exposed in the code as constants?

Reply:
Good catch. I forgot to add the Bitflow variables to the [playwright.yml](https://github.com/leather-io/extension/pull/5860/files#diff-7afcd2d8f7b49bda74843f209eefb7b2da45f7e7803bf2e4bd636699b76aa2d3) workflow so the integration tests all broke.

Just added them and tests are passing now.